bundles/org.corpus_tools.hexatomic.graph/src/main/java/org/corpus_tools/hexatomic/graph/GraphEditor.java

```diff
@@ -485,7 +518,8 @@ private void updateGraphViewer(SDocumentGraph graph, boolean scrollToFirstToken)
 
     if (scrollToFirstToken) {
       viewer.getGraphControl().getRootLayer().setScale(0.0);
-      // We can only scroll to the first token after the layout has been applied, which can be
+      // We can only scroll to the first token after the layout has been applied,
+      // which can be
       // asynchronous
```

Comment:
```suggestion
      // which can be asynchronous
```

Minor formatting fix